flexget/plugins/modify/manipulate.py

```diff
@@ -117,12 +119,18 @@ def process(self, entry, jobs):
                     if not field_value:
                         log.warning('Cannot extract, field `%s` is not present' % from_field)
                         continue
-                    match = re.search(config['extract'], field_value, re.I | re.U)
-                    if match:
-                        groups = [x for x in match.groups() if x is not None]
-                        log.debug('groups: %s' % groups)
-                        field_value = config.get('separator', ' ').join(groups).strip()
+                    if config.get('findall'):
+                        match = re.findall(config['extract'], field_value)
+                        log.debug('all matches: %s' % match)
```

Comment:
Use logger string interpolation
```suggestion
                        log.debug('all matches: %s', match)
```

Reply:
I guess then it might be worth changing all log.debug in this file to that notation then. As all the others use '%' hence why I used it there.